src/Storefront/Resources/views/storefront/component/checkout/offcanvas-cart-summary.html.twig

```diff
@@ -30,7 +30,7 @@
                         </span>
 
                         <span class="col-5 pb-2 shipping-value shipping-cost">
-                            <strong>+ {{ activeShipping.shippingCosts.totalPrice|currency }}</strong>
+                            <strong>{{activeShipping.shippingCosts.totalPrice<0?'- ':'+ '}}{{ activeShipping.shippingCosts.totalPrice|abs|currency }}</strong>
```

Comment:
What about other places? Places that come to my mind:
- Everything related to the storefront - Checkout / Offancas cart / Confirm Page
- Administration order detail page
- generated documents: invoice etc.
Can you check if the change is needed also at the other places?